src/index.js

```diff
@@ -35,19 +35,18 @@ export default class ReactSVG extends Component {
   renderSVG(props = this.props) {
     const {callback: each, className, evalScripts, path, style} = props;
 
-    const div = document.createElement('div');
-    div.innerHTML = ReactDOMServer.renderToStaticMarkup(
-      <div>
-        <div className={className} data-src={path} style={style} />
-      </div>
+    const wrapper = document.createElement('div');
+    ReactDOM.render(
+      <div className={className} data-src={path} style={style} />,
+      wrapper,
+      () => {
+        SVGInjector(wrapper.firstChild, {
+          evalScripts,
+          each,
+        });
+        this.container.appendChild(wrapper.firstChild);
```

Comment:
Any reason why the order of this has been flipped compared to the original version? I vaguely remember the original order was deliberate too but I'll have to check the issues again to be sure.